tests/features/common/actions/table.action.js

```diff
@@ -0,0 +1,63 @@
+import { expect } from 'chai'
+
+function getColumnValues(driver, table, columnName) {
+  return driver
+    .findElements(table.tableColumns[columnName])
+    .then(async function(elements) {
+      return await Promise.all(
+        elements.map(async function(element) {
+          return await element.getText()
+        })
+      )
+    })
+}
```

Comment:
Using `async`/`await` style:
```suggestion
function async getColumnValues(driver, table, columnName) {
  const elements = await driver.findElements(table.tableColumns[columnName])
  return Promise.all(elements.map(element => element.getText()))
}
```
**or** using promise style:
```suggestion
function getColumnValues(driver, table, columnName) {
  return driver
    .findElements(table.tableColumns[columnName])
    .then(function(elements) {
      return Promise.all(
        elements.map(element => element.getText())
      )
    })
}
```